docs/content/querying/dimensionspecs.md

```diff
@@ -201,3 +201,28 @@ A null dimension value can be mapped to a specific value by specifying the empty
 This allows distinguishing between a null dimension and a lookup resulting in a null.
 For example, specifying `{"":"bar","bat":"baz"}` with dimension values `[null, "foo", "bat"]` and replacing missing values with `"oof"` will yield results of `["bar", "oof", "baz"]`.
 Omitting the empty string key will cause the missing value to take over. For example, specifying `{"bat":"baz"}` with dimension values `[null, "foo", "bat"]` and replacing missing values with `"oof"` will yield results of `["oof", "oof", "baz"]`.
+
+### Lookup extraction function (EXPERIMENTAL)
```

Comment:
IMO there's nothing wrong with a feature being experimental for the first release it's in, if there's a good reason. Good reasons might include it's not had a lot of testing yet, or it's believed that the interfaces might change before being finalized.